src/rules/javascript/webextension_api.js

```diff
@@ -0,0 +1,16 @@
+import { DEPRECATED_APIS } from 'const';
+import { apiToMessage } from '../../utils';
+
+export function webextension_api(context) {
+  return {
+    MemberExpression: function(node) {
+      if (node.object.object &&
```

Comment:
You could be more defensive here if `node.object` is undefined this will blow up.